vllm/engine/async_llm_engine.py

```diff
@@ -526,11 +529,19 @@ async def add_request(
                 if shortened_token_ids is not None:
                     shortened_token_ids = shortened_token_ids[:self.
                                                               max_log_len]
+
+            logger_data = {
+                "request": request_id,
+                "prompt": shortened_prompt,
+                "sampling_params": sampling_params,
+                "prompt_token_ids": shortened_token_ids,
+                "lora_request": lora_request,
+            }
```

Comment:
```suggestion
            logger_data = {
                "event": "request-received",
                "lora_request": lora_request,
                "prompt": shortened_prompt,
                "prompt_token_ids": shortened_token_ids,
                "request": request_id,
                "sampling_params": sampling_params,
            }
```

Reply:
This seems like it's probably a useful use of `logger_data`; an enterprise would want to potentially monitor some of these request metrics. I wonder if some counts might make sense instead of raw values here.